Author: Jean-Louis Martineau <JMartineau@carbonite.com>
Description: Fix a problem found with samba on Debian 10 aka buster, part 2.

Found on mailing-list amanda-user:

Add the same pattern near line 757:
   return if $line =~ /^Unable to initialize messaging context/;

(...)

Index: amanda.git/application-src/amsamba.pl
===================================================================
--- amanda.git.orig/application-src/amsamba.pl	2019-07-17 20:54:02.369149536 +0100
+++ amanda.git/application-src/amsamba.pl	2019-07-17 21:09:58.884796126 +0100
@@ -760,6 +760,7 @@ sub command_backup {
 	return if $line =~ /tarmode is now /;
 	return if $line =~ /tar_re_search set/;
 	return if $line =~ /WARNING/g;
+	return if $line =~ /^Unable to initialize messaging context/;
 	if ($line =~ /dumped (\d+) files and directories/) {
 	    $nb_files = $1;
 	    return;
